<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientsMenu</class>
 <widget class="QWidget" name="ClientsMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>833</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>833</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Управление клиентами</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
  </property>
  <widget class="QLabel" name="MainTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>126</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="text">
    <string>Клиенты</string>
   </property>
  </widget>
  <widget class="QTableView" name="ClientTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>771</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="AddClient">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>138</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить клиента</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveClient">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>520</y>
     <width>138</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить клиента</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="AdressLayout">
    <property name="spacing">
     <number>41</number>
    </property>
    <item>
     <widget class="QLabel" name="AdressLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Адрес</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="AdressEdit">
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PhoneLayout">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <widget class="QLabel" name="PhoneLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PhoneEdit">
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="placeholderText">
       <string>+7 XXX XXX XX XX</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>460</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RegLayout">
    <property name="spacing">
     <number>25</number>
    </property>
    <item>
     <widget class="QLabel" name="RegLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Рег Номер</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="RegEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>281</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="FIOLayout">
    <property name="spacing">
     <number>46</number>
    </property>
    <item>
     <widget class="QLabel" name="FIOLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ФИО</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="FIOEdit">
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
